doc/guide/feature-selinux.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+	"http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="feature-selinux">
+  <title>SELinux Policy</title>
+
+  <para>If present on the system Cockpit can enable and disable SELinux. It can also
```

Comment:
Technically this isn't correct. We can only change the mode - between enforcing and permissive. We explicitly didn't want Cockpit to influence the overall state (enabled, disabled), since a change there requires a reboot and can break systems.